scripts/sync_makefiles.sh

```diff
@@ -15,59 +15,85 @@ orgs="prometheus prometheus-community"
 
 GITHUB_TOKEN="${GITHUB_TOKEN:-}"
 if [ -z "${GITHUB_TOKEN}" ]; then
-	echo -e "\e[31mGitHub token (GITHUB_TOKEN) not set. Terminating.\e[0m"
-	exit 1
+  echo -e "\e[31mGitHub token (GITHUB_TOKEN) not set. Terminating.\e[0m"
+  exit 1
 fi
 
 # Go to the root of the repo
-cd "$(git rev-parse --show-cdup)"
+cd "$(git rev-parse --show-cdup)" || exit 1
 
 source_makefile="$(pwd)/Makefile.common"
 source_checksum="$(sha256sum Makefile.common | cut -d' ' -f1)"
 
-tmp_dir=$(mktemp -d)
-trap "rm -rf ${tmp_dir}" EXIT
+tmp_dir="$(mktemp -d)"
+trap 'rm -rf "${tmp_dir}"' EXIT
```

Comment:
${tmp_dir:?}

Reply:
I don't think this is necessary as the whole script is `set -u`.